components/automate-ui/src/app/pages/project/details/project-details.component.html

```diff
@@ -78,7 +78,7 @@
                   <a [routerLink]="['/settings', 'projects', project?.id, 'rules', rule.id]">{{ rule.name }}</a>
                 </chef-td>
                 <chef-td>{{ rule.type | titlecase }}</chef-td>
-                <chef-td>{{ rule.conditions.length }} conditions</chef-td>
+                <chef-td>{{ getConditionsText(rule.conditions.length) }}</chef-td>
```

Comment:
Please use the `pluralize` pipe to get the plural here. That way it obviates the need for a custom function every time.

```suggestion
                <chef-td>{{ rule.conditions.length | pluralize : 'condition' : 's' }}  </chef-td>
```

Other example in the code base: https://github.com/chef/automate/blob/98cfe6344fab6dd9636317e0ce2dcc2128341643/components/automate-ui/src/app/pages/policy/add-members/policy-add-members.component.html#L47